### PR TITLE
chore(pr-template): update PR template to include Design QA requirement

### PR DESCRIPTION
**Pull request by monicawheeler**
# Description

This PR updates the Pull Request template to include a Design QA requirement section. This addition ensures that all future PRs consider the need for design review and approval before merging.

## Type of change

- [x] Chore (Other changes that don't modify src or test files)

# How Has This Been Tested?

This change has been tested by previewing the updated PR template.

**Test Configuration**:
N/a

# Checklist:

- [ ]  My code follows the style guidelines of this project
- [x] I have performed a self-review of my code (change)
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ]  I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ]  I have added tests that prove my fix is effective or that my feature works
- [ ]  New and existing tests pass locally with my changes

### CODE REVIEW

**Review by goodwinchris — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -29,18 +29,19 @@ Please also list any relevant details for your test configuration.
...
+- Screen readers:
+- Misc:
 
 # Checklist:
 
 - [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
-- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have performed a self-review of my code
+- [ ] I have commented on my code, particularly in hard-to-understand areas
```
Comment (ju-Skinner) [inline .github/pull_request_template.md]: I believe it was better the other way. I comment my code, as I don't comment on my code.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -29,18 +29,19 @@ Please also list any relevant details for your test configuration.
...
+- Sage versions:
+- OS:
+- Browsers:
+- Screen readers:
+- Misc:
 
 # Checklist:
 
 - [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
```
Comment (ju-Skinner) [inline .github/pull_request_template.md:41]: The PR itself implies ownership.

**Review by ju-Skinner — CHANGES_REQUESTED** [review]
Rejection is due to "comment" vs "comment on"

File: .github/pull_request_template.md (hunk 1 above)
Comment (monicawheeler) [reply .github/pull_request_template.md]: I will blame Grammarly for yelling at me for this one but secondarily then blame myself for not catching it. 
In any event, I agree, changing it back.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -12,13 +12,13 @@ Please delete options that are not relevant.
 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
```
Comment (monicawheeler) [inline .github/pull_request_template.md]: This one is going back to the original as well. Thanks @ju-Skinner

**Review by ju-Skinner — APPROVED** [review]
Thank you

**Pull request merged**